### PR TITLE
M4a linux issues 36

### PR DESCRIPTION
**Pull request by stevenschlansker**
Hi @the-m4a , I am trying to run CQUI on Linux (via Proton) and noticed that nothing works - but things seem to work better once this PR is merged in!  I resolved the conflicts by just deleting the delete/conflict.

I have not yet tested all features, but without this PR *nothing* works.
Can we get this merged in at some point so Linux users can play too? :) Thanks!

### CODE REVIEW

**Comment by stevenschlansker** [comment]
Ok, sorry to follow up so quickly, but while this PR fixes minimap and UI elements, upgraded city view (name, production select, etc) is totally missing on the map -- you can't even click your own cities.

**Comment by the-m4a** [comment]
Yep - actually fired up my Linux VM (via windows for Linux) again last night in an attempt to get things going.  Once I get to the point where things load without error I will need actual play testers, the WSL Linux *barely* runs Civ so playing any more than a few turns is brutal

**Comment by stevenschlansker** [comment]
I volunteer to play test on Linux!  I'm very interested in getting this working - I have bare basic Lua skills (from hacking on Factorio mods) but zero Civ modding experience :)

**Comment by the-m4a** [comment]
@stevenschlansker I will publish my current working branch on Linux - I got the minimap to load, the changes are similar to what's in here but not completely.  Go over to the tracking thread and we can discuss there
https://github.com/civfanatics/CQUI_Community-Edition/issues/203

File: Assets/Expansion1/Replacements/worldtracker_expansion1.lua [hunk 1]
```diff
@@ -1,3 +1,6 @@
+-- TEMP
```
Comment (the-m4a) [inline Assets/Expansion1/Replacements/worldtracker_expansion1.lua:1]: making a note here as I appreciate this work but want to focus on a single branch if we can, so this note exists only to block the PR until we figure out issues and determine if this should be merged in (I don't want this to be merged before we all figure it out)

**Review by the-m4a — CHANGES_REQUESTED** [review]

**Comment by enderger** [comment]
I have tested this PR on my NixOS 21.03 box and it works fine. The only issue I found was that R&F city banners don't load, however that is not an issue in Gathering Storm.

**Pull request closed without merging**